### ISSUE TITLE
A layer font to colr font converter

### ISSUE THREAD

**Issue by davelab6** [googlefonts/color-fonts#103]
There are a few "layer fonts" in gf and many outside, and a converter would be useful to spur adoption

**Comment by rsheeter** [comment]
What do you mean by "layer font"? Can you point out examples?

**Comment by applecuckoo** [comment]
@rsheeter My guess is something like DJR Bungee, I think the exact term is "chromatic layering". See the Bungee page about it [here](https://github.com/djrrb/Bungee/blob/master/documentation/2-chromatic-layers.md).

**Comment by rsheeter** [comment]
Ty for the reminder. We did Bungee, https://fonts.google.com/?query=Bungee+Spice. @davelab6 are there other examples we should look at?

**Comment by applecuckoo** [comment]
Not quite, the original Bungee came with a 'layered' set where you could do different color layers, with the different fonts (inline, regular, outline, shade), and then hacking them on top of each other by turning off the linespacing. I asked about this on the Bungee repo, and @justvanrossum replied saying that there was something in development based on this idea. (If you want to see the issue, it's https://github.com/djrrb/Bungee/issues/82.
